todo_app/data/trello_items.py

```diff
@@ -5,7 +5,7 @@ def __init__(self, id, name, status):
         self.id = id
         self.name = name
         self.status = status
-    
+
     @classmethod
     def from_trello_card(cls, card):
         return cls(card['id'], card['name'], card['idList'])
```

Comment:
The final bit I was missing when integrating the ViewModel was here - you're currently setting the `status` field using the _id_, but you're filtering with the _name_, so we need to bring those into line. Probably the name is more user-friendly, so we could:
* Change this method to extract a new field from card, e.g. `listName`:
```suggestion
    def from_trello_card(cls, card):
        return cls(card['id'], card['name'], card['listName'])
```

And as long as we set that field on the card before we call this method, we should be OK, e.g. in your loop around line 35:
```python
    for trello_list in todolist_json:
        for card in trello_list['cards']:
            # we add the list name here so it can be used, we could alternatively pass it through as a separate argument
            card['listName'] = trello_list['name']
            cards.append(Item.from_trello_card(card))
```